provision/pull-images.sh

```diff
@@ -54,8 +54,8 @@ if [ -z "${NAME_PREFIX}" ]; then
         gcr.io/google-samples/gb-frontend:v6 \
         gcr.io/google_samples/gb-redis-follower:v2 \
         quay.io/cilium/cilium-envoy:63de0bd958d05d82e2396125dcf6286d92464c56 \
-        quay.io/cilium/cilium-builder:2021-01-20 \
-        quay.io/cilium/cilium-runtime:2021-01-20 \
+        quay.io/cilium/cilium-builder:2021-02-15 \
+        quay.io/cilium/cilium-runtime:2021-02-15 \
         quay.io/cilium/hubble:v0.7.1 \
```

Comment:
fwiw: This can now be removed. We are now downloading Hubble from the GitHub releases since the switch to the `images/` build

Reply:
Nice, thanks for the heads-up. Will remove `quay.io/cilium/hubble` from the pre-pulled image as well.

Reply:
Done